core/src/main/java/com/linecorp/armeria/server/AbstractHttpResponseHandler.java

```diff
@@ -82,6 +85,14 @@ void disconnectWhenFinished() {
     }
 
     final boolean tryComplete(@Nullable Throwable cause) {
+        if (req.isHttp1WebSocket()) {
```

Comment:
I'm concerned `decreaseActive*Requests` will never have a chance to be called.
e.g. If a request is received but a connection is immediately closed, will `decreaseActive*Requests` always be called?

I'm wondering if the future where the request/response is completed is a more fail-safe location. What do you think @minwoox ? 

https://github.com/line/armeria/blob/cf45ab6e8546e2398ebebdda78bda304000548b5/core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java#L768